compiler/plugins/target/ROCM/ROCMTarget.cpp

```diff
@@ -36,25 +34,31 @@
 #include "mlir/Target/LLVMIR/Dialect/ROCDL/ROCDLToLLVMIRTranslation.h"
 #include "mlir/Target/LLVMIR/Export.h"
 
-static llvm::cl::opt<std::string>
-    clROCMTargetChip("iree-rocm-target-chip",
-                     llvm::cl::desc("ROCm target Chip"),
-                     llvm::cl::init("gfx908"));
-
-static llvm::cl::opt<bool>
-    clROCMLinkBC("iree-rocm-link-bc",
-                 llvm::cl::desc("Whether to try Linking to AMD Bitcodes"),
-                 llvm::cl::init(false));
-
-static llvm::cl::opt<std::string> clROCMBitcodeDir(
-    "iree-rocm-bc-dir", llvm::cl::desc("Directory of ROCM Bitcode"),
-    llvm::cl::init(mlir::iree_compiler::findPlatformLibDirectory("rocm")));
-
 namespace mlir {
 namespace iree_compiler {
 namespace IREE {
 namespace HAL {
 
+struct ROCMOptions {
+  std::string targetChip;
+  bool linkBitcode;
+  std::string bitcodeDirectory;
```

Comment:
I think these options aren't getting set... might have misconfigured something. Should also check the other compiler target plugins.